pwa/src/components/logs/logModal/LogModal.tsx

```diff
@@ -16,26 +16,16 @@ interface LogModalProps {
 
 const LogModal: React.FC<LogModalProps> = ({ log }) => {
   const API: APIService = React.useContext(APIContext);
-  const [callIdLogs, setLogs] = React.useState([]);
-  const [showSpinner, setShowSpinner] = React.useState(false);
-
-  React.useEffect(() => {
-    handleSetOutgoingLogs();
-  }, [API]);
+  const [callIdLogs, setCallIdLogs] = React.useState(null);
```

Comment:
To make this name even more descriptive, we could rename it to `outgoingCallIdLogs`.